english/Rhyme.txt

```diff
@@ -0,0 +1,5 @@
+Roses are red, my name's not dave,
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.